### DISCUSSION TITLE
when to support Optional chaining (?.)

### DISCUSSION THREAD

**Discussion by xiaodi0003** [eslint/eslint#14781] [Q&A]
it looks like that eslint do not suport  Optional chaining (?.) just now

**Comment by aladdin-add** [comment]
optional chaining has been supported since eslint v7.5.0.  https://github.com/eslint/eslint/commit/6ea3178776eae0e40c3f5498893e8aab0e23686b

you will be able to config `ecmaVerson: 2020` to use it.
